bindings/python/pymongoarrow/schema.py

```diff
@@ -55,6 +55,9 @@ def __init__(self, schema):
     def __iter__(self):
         yield from self.typemap
 
+    def __repr_(self):
+        return self.typemap
+
```

Comment:
Accidental addition?

Reply:
I just noticed a typo there, but it's intentional. Given the importance of these schema, we need to be able to print them meaningfully. This seems to fit under the heading "docs clean up". What do you think?